docs/xxopenj9commandlineenv.md

```diff
@@ -0,0 +1,40 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]OpenJ9CommandLineEnv
+
+This option controls whether the VM captures the command line in the environment variable `OPENJ9_JAVA_COMMAND_LINE`.
+
+## Syntax
+
+        -XX:[+|-]OpenJ9CommandLineEnv
+
+| Setting                     | Effect  |
+|-----------------------------|---------|
+| `-XX:+OpenJ9CommandLineEnv` | Enable  |
```

Comment:
Other options put an indication beside the default option (`:fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>`)

Reply:
I think we should do that and indicate the platform variation in brackets. Also we could mention that enabling it on z/OS may lead to undesirable results.

Reply:
Yes, but I wasn't sure how to display that the default varies by platform. How about *two* additional columns (for z/OS and other platforms) like this?

| Setting                     | Effect  | Default on z/OS | Default on Other Platforms |
|-----------------------------|---------|:-:|:-:|
| `-XX:+OpenJ9CommandLineEnv` | Enable  |   | x |
| `-XX:-OpenJ9CommandLineEnv` | Disable | x |   |

Reply:
It's fine with me. @doveye may also have an opinion.

Reply:
I think using separate columns is clearer, and there's precedence for that approach: https://www.eclipse.org/openj9/docs/xxidletuningcompactonidle/.